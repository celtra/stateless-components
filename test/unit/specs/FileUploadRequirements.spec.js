import FileUploadRequirements from '@/stateless/file_upload_requirements.vue'
import snapshotMixin from '../component_snapshot_mixin'

describe('FileUploadRequirements', () => {
    snapshotMixin(FileUploadRequirements)
})
