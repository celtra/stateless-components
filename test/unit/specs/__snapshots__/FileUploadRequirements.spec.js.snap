// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileUploadRequirements should match the snapshot 1`] = `
<div
  class="file-upload-requirements"
>
  <p
    class="file-upload-requirements__requirement"
  >
    
        Format:
        
    <span
      class="file-upload-requirements__requirement-value"
    >
      PNG, JPG
    </span>
  </p>
  <p
    class="file-upload-requirements__requirement"
  >
    
        Size:
        
    <span
      class="file-upload-requirements__requirement-value"
    >
      max. 3MB
    </span>
  </p>
</div>
`;
